### DISCUSSION TITLE
CompreFace 1.0.1

### DISCUSSION THREAD

**Discussion by pospielov** [exadel-inc/CompreFace#846] [General]
## Features

* Updated CUDA to 11.1 version (Ampere Nvidia GPU generation is supported)

## Migration from 1.0.0 version
* Please check this article to make sure your GPU is supported. If you use the CPU version, no actions are needed.

## How to install:
1. Download CompreFace_1.0.1.zip archive or run:
`wget -q -O tmp.zip 'https://github.com/exadel-inc/CompreFace/releases/download/v1.0.1/CompreFace_1.0.1.zip' && unzip tmp.zip && rm tmp.zip`
2. To start CompreFace run:
`docker-compose up -d`
3. Open in your browser: http://localhost:8000/login
4. To stop properly run: 
`docker-compose stop`
5. To start again run: 
`docker-compose start`

More detail install guide(and Windows instructions) is on our [readme](https://github.com/exadel-inc/CompreFace#getting-started-with-compreface)﻿ page.

<hr /><em>This discussion was created from the release <a href='https://github.com/exadel-inc/CompreFace/releases/tag/v1.0.1'>CompreFace 1.0.1</a>.</em>

**Comment by chrisborell** [comment]
Hello! I'm have a NVIDIA GPU running CompreFace with Desktop Docker. How can i check or make it use the GPU? Currently it is still using the CPU with 1.0.1 installed. Any advice would be great! Thanks!

**Reply by pospielov** [reply]
Default CompreFace build doesn't support GPU. But each release contains `custom-builds` folder. You can find GPU builds among them.
Please look at our documentation about custom builds:
https://github.com/exadel-inc/CompreFace/blob/master/docs/Custom-builds.md
